lib/media/segment_prefetch.js

```diff
@@ -147,7 +147,12 @@ shaka.media.SegmentPrefetch = class {
     }
 
     // init segments are ignored from the prefetch limit
-    if (!this.initSegmentPrefetchMap_.has(initSegmentReference)) {
+    const initSegments = Array.from(this.initSegmentPrefetchMap_.keys());
+    const someReference = initSegments.find((reference) => {
```

Comment:
Would this be better to use `some` here? Since we don't actually need the ref only whether we have it.